src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java

```diff
@@ -54,6 +54,7 @@ public final class ModificationUtils {
     public static final String CONNECTION_NAME_FIELD_NAME = "Connection name";
     public static final String CONNECTION_DIRECTION_FIELD_NAME = "Connection direction";
     public static final String CONNECTION_POSITION_FIELD_NAME = "Connection position";
+    public static final String NETWORK_NOT_EXIST = " does not exist in network";
```

Comment:
well this is not really that, maybe more like NOT_EXIST_IN_NETWORK

Reply:
okey